src/index.css

```diff
@@ -1,27 +1,4 @@
 body {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    "Segoe UI",
-    "Roboto",
-    "Oxygen",
-    "Ubuntu",
-    "Cantarell",
-    "Fira Sans",
-    "Droid Sans",
-    "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    "Courier New",
-    monospace;
+  background-color: var(--surface-100);
```

Comment:
- [ ] please kindly ensure that the styling is almost exact like the Zeplin version. nice job. please the add book button should be aligned with the page like in the original version.
yours
![Screenshot (848)_LI](https://user-images.githubusercontent.com/65298048/222548669-92e6aa67-2896-4e47-9676-ed51c02bb300.jpg)
Zeplin
![Screenshot (847)_LI](https://user-images.githubusercontent.com/65298048/222548925-9960c24b-078c-4c91-a1e7-d5937498cfbc.jpg)

Reply:
thanks  updated based on the feedback